src/main/java/mate/academy/springbootintro/service/BookServiceImpl.java

```diff
@@ -22,22 +24,39 @@ public class BookServiceImpl implements BookService {
     public BookDto createBook(CreateBookRequestDto createBookRequestDto) {
         Book book = bookMapper.toModel(createBookRequestDto);
         book.setIsbn("abc" + new Random().nextInt(1000));
-        Book savedBook = bookRepository.createBook(book);
+        Book savedBook = bookRepository.save(book);
         return bookMapper.toDto(savedBook);
     }
 
     @Override
     public List<BookDto> getAll() {
-        return bookRepository.getAll().stream()
+        return bookRepository.findAll().stream()
                 .map(bookMapper::toDto)
                 .toList();
     }
 
     @Override
     public BookDto getBookById(Long id) {
-        Book book = bookRepository.getBookById(id).orElseThrow(
+        Book book = bookRepository.findById(id).orElseThrow(
                 () -> new EntityNotFoundException("Can't find book by id: " + id));
         return bookMapper.toDto(book);
 

```

Comment:
Multiple unnecessary new lines - your code doesn't look consistent if you sometimes put new line after return and sometimes you don't, in one method lines are separated and in the other they are not.